build.sbt

```diff
@@ -89,6 +87,7 @@ lazy val commonDependencies = Seq(
   "com.typesafe" % "config" % "1.4.2",
   scalatest % "test",
   "com.typesafe.scala-logging" % "scala-logging_2.13" % "3.9.5",
+  "ch.qos.logback" % "logback-classic" % "1.4.14",
```

Comment:
everything that sends logs needs this at runtime, otherwise you just get a "NOP logger" message and no actual logging.  This is mostly relevant for running tests against subprojects, as we had manually added it to all the actual deployed projects.